config/core/configmaps/features.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-features
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
+
+data:
+  _example: |
+    ################################
+    #                              #
+    #    EXAMPLE CONFIGURATION     #
+    #                              #
+    ################################
+
+    # This block is not actually functional configuration,
+    # but serves to illustrate the available configuration
+    # options and document them in a way that is accessible
+    # to users that `kubectl edit` this config map.
+    #
+    # These sample configuration options may be copied out of
+    # this example block and unindented to be in the data block
+    # to actually change the configuration.
+
+    # Indicates whether multi container support is enabled
+    enable-multi-container: "false"
```

Comment:
This is different from the format described in the proposal (c.f. "naming" and "scope"). Also I was wondering if the intention is to have a separate ConfigMap for each core component (serving and eventing)? If that's the case, the prefix `serving.knative.dev/feature/` can probably be removed from the ConfigMap key.

Feel free to ping me in the proposal if you prefer to discuss it there :)

Reply:
Right, for this existing flag, I kept the original name for now, but yes we should probably add a prefix. Regarding eventing vs serving, I think it will be separate ConfigMap since they live in different namespaces.

Reply:
Yeah, I think scoping things by group would be helpful to avoid collisions, I'd forgotten we discussed this on the design 😅